src/components/RadioCard/styles.scss

```diff
@@ -0,0 +1,40 @@
+.radio-card {
```

Comment:
Vet ikke helt hvordan dere har gjort scss styling eller har blitt enige om, men et lite tips til hvordan vi har gjort det tidligere er:

```
.radio-card {
    ...

    &__description {...}

    &:hover {...}

    &--selected {...}
}
```

Her er det da entydig at disse "egenskapene" tilhører `radio-card`. Litt finere enn å definere .radio-card:hover for seg seg.

Det vi gjør når f.eks. vi har en "hoved" div med en `className` så vil classNames under denne tilhøre denne "hoved"-div'en. E.g. 

```
<div className="main">
    <div className="main__content" />
</div>
```

Når en scss fil vokser i størrelsen så vil det bli litt bedre lesbarhet også :)

Reply:
Personlig synes jeg det er _mindre_ lesbart, men det gjør meg ingenting å endre det! :) Det er tross alt andre som skal ta over prosjektet også, så det er greit å følge konvensjoner